### PR TITLE
Updating 2.0-2.4, 2.5 with new docs structure

### PR DESCRIPTION
**Pull request by jtravee**
Updating 2.0 - 2.4, 2.5, and 2.6 with new content.

### CODE REVIEW

**Review by btat — CHANGES_REQUESTED** [review]
In the test deployment (https://github.com/rancher/rancher-docs/runs/8256706580?check_suite_focus=true), there are still some broken links being introduced in this PR.

- `Warning:  Docs markdown link couldn't be resolved: (../getting-started/introduction/what-are-divio-docs?.md) in /home/runner/work/rancher-docs/rancher-docs/versioned_docs/version-2.5/pages-for-subheaders/introduction.md for version 2.5`
- `Warning:  Docs markdown link couldn't be resolved: (../getting-started/introduction/what-are-divio-docs?.md) in /home/runner/work/rancher-docs/rancher-docs/docs/pages-for-subheaders/introduction.md for version current`
- `Warning:  Docs markdown link couldn't be resolved: (../getting-started/introduction/what-are-divio-docs?.md) in /home/runner/work/rancher-docs/rancher-docs/versioned_docs/version-2.0-2.4/pages-for-subheaders/introduction.md for version 2.0-2.4`
- `Docs markdown link couldn't be resolved: (workloads-and-pods.md) in /home/runner/work/rancher-docs/rancher-docs/versioned_docs/version-2.5/reference-guides/cli-with-rancher/rancher-cli.md for version 2.5`
- `Warning:  Docs markdown link couldn't be resolved: (../how-to-guides/advanced-user-guides/manage-clusters/access-clusters/use-kubectl-and-kubeconfig.md) in /home/runner/work/rancher-docs/rancher-docs/docs/reference-guides/rancher-manager-architecture/communicating-with-downstream-user-clusters.md for version current`
- `On source page path = /reference-guides/rancher-manager-architecture/communicating-with-downstream-user-clusters:
   -> linking to ../how-to-guides/advanced-user-guides/manage-clusters/access-clusters/use-kubectl-and-kubeconfig.md (resolved as: /reference-guides/how-to-guides/advanced-user-guides/manage-clusters/access-clusters/use-kubectl-and-kubeconfig.md)`

**Comment by jtravee** [comment]
> In the test deployment (https://github.com/rancher/rancher-docs/runs/8256706580?check_suite_focus=true), there are still some broken links being introduced in this PR.
> 
> * `Warning:  Docs markdown link couldn't be resolved: (../getting-started/introduction/what-are-divio-docs?.md) in /home/runner/work/rancher-docs/rancher-docs/versioned_docs/version-2.5/pages-for-subheaders/introduction.md for version 2.5`
> * `Warning:  Docs markdown link couldn't be resolved: (../getting-started/introduction/what-are-divio-docs?.md) in /home/runner/work/rancher-docs/rancher-docs/docs/pages-for-subheaders/introduction.md for version current`
> * `Warning:  Docs markdown link couldn't be resolved: (../getting-started/introduction/what-are-divio-docs?.md) in /home/runner/work/rancher-docs/rancher-docs/versioned_docs/version-2.0-2.4/pages-for-subheaders/introduction.md for version 2.0-2.4`
> * `Docs markdown link couldn't be resolved: (workloads-and-pods.md) in /home/runner/work/rancher-docs/rancher-docs/versioned_docs/version-2.5/reference-guides/cli-with-rancher/rancher-cli.md for version 2.5`
> * `Warning:  Docs markdown link couldn't be resolved: (../how-to-guides/advanced-user-guides/manage-clusters/access-clusters/use-kubectl-and-kubeconfig.md) in /home/runner/work/rancher-docs/rancher-docs/docs/reference-guides/rancher-manager-architecture/communicating-with-downstream-user-clusters.md for version current`
> * `On source page path = /reference-guides/rancher-manager-architecture/communicating-with-downstream-user-clusters: -> linking to ../how-to-guides/advanced-user-guides/manage-clusters/access-clusters/use-kubectl-and-kubeconfig.md (resolved as: /reference-guides/how-to-guides/advanced-user-guides/manage-clusters/access-clusters/use-kubectl-and-kubeconfig.md)`

@btat

**Comment by btat** [comment]
> @btat

For the first few, the question mark is causing the link issue.

**Comment by jtravee** [comment]
> > @btat
> 
> For the first few, the question mark is causing the link issue.

Thanks

**Review by btat — CHANGES_REQUESTED** [review]

File: docs/pages-for-subheaders/cis-scans.md [hunk 1]
```diff
@@ -11,8 +11,9 @@ The `rancher-cis-benchmark` app leverages <a href="https://github.com/aquasecuri
 - [About the Generated Report](#about-the-generated-report)
 - [Test Profiles](#test-profiles)
 - [About Skipped and Not Applicable Tests](#about-skipped-and-not-applicable-tests)
-- [Roles-based Access Control](../explanations/integrations-in-rancher/cis-scans/rbac-for-cis-scans.md)
-- [Configuration](../explanations/integrations-in-rancher/cis-scans/configuration-reference.md)
+- [Role-based Access Control](#role-based-access-control)
```
[the remark this answers is not in the document]
Comment (jtravee) [reply docs/pages-for-subheaders/cis-scans.md]: Updated per feedback.

File: docs/pages-for-subheaders/fleet-gitops-at-scale.md [hunk 2]
```diff
@@ -1 +1,72 @@
...
\ No newline at end of file
+---
+title: Fleet - GitOps at Scale
+---
+
+Fleet is GitOps at scale. Fleet is designed to manage up to a million clusters. It’s also lightweight enough that it works great for a [single cluster](https://fleet.rancher.io/single-cluster-install/) too, but it really shines when you get to a [large scale](https://fleet.rancher.io/multi-cluster-install/). By large scale we mean either a lot of clusters, a lot of deployments, or a lot of teams in a single organization.
+
+Fleet is a separate project from Rancher, and can be installed on any Kubernetes cluster with Helm.
+
+- [Architecture](../explanations/integrations-in-rancher/fleet-gitops-at-scale/architecture.md)
```
[the remark this answers is not in the document]
Comment (jtravee) [reply docs/pages-for-subheaders/fleet-gitops-at-scale.md]: Updated per feedback.

File: docs/pages-for-subheaders/fleet-gitops-at-scale.md [hunk 3]
```diff
@@ -1 +1,72 @@
...
+- [Accessing Fleet in the Rancher UI](#accessing-fleet-in-the-rancher-ui)
+- [Windows Support](../explanations/integrations-in-rancher/fleet-gitops-at-scale/windows-support.md)
+- [GitHub Repository](#github-repository)
+- [Use Fleet Behind a Proxy](../explanations/integrations-in-rancher/fleet-gitops-at-scale/use-fleet-behind-a-proxy.md)
```
[the remark this answers is not in the document]
Comment (jtravee) [reply docs/pages-for-subheaders/fleet-gitops-at-scale.md]: Updated per feedback.

File: versioned_docs/version-2.0-2.4/how-to-guides/advanced-user-guides/cis-scan-guides/schedule-recurring-scans.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+---
```
[the remark this answers is not in the document]
Comment (jtravee) [reply versioned_docs/version-2.0-2.4/how-to-guides/advanced-user-guides/cis-scan-guides/schedule-recurring-scans.md]: Updated per feedback.

File: versioned_docs/version-2.0-2.4/how-to-guides/advanced-user-guides/cis-scan-guides/set-alerts.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
+---
```
[the remark this answers is not in the document]
Comment (jtravee) [reply versioned_docs/version-2.0-2.4/how-to-guides/advanced-user-guides/cis-scan-guides/set-alerts.md]: Updated per feedback.

File: docs/pages-for-subheaders/fleet-gitops-at-scale.md [hunk 6]
```diff
@@ -1 +1,72 @@
...
+title: Fleet - GitOps at Scale
+---
+
+Fleet is GitOps at scale. Fleet is designed to manage up to a million clusters. It’s also lightweight enough that it works great for a [single cluster](https://fleet.rancher.io/single-cluster-install/) too, but it really shines when you get to a [large scale](https://fleet.rancher.io/multi-cluster-install/). By large scale we mean either a lot of clusters, a lot of deployments, or a lot of teams in a single organization.
+
+Fleet is a separate project from Rancher, and can be installed on any Kubernetes cluster with Helm.
+
+- [Architecture](../explanations/integrations-in-rancher/fleet-gitops-at-scale/architecture.md)
+- [Accessing Fleet in the Rancher UI](#accessing-fleet-in-the-rancher-ui)
+- [Windows Support](../explanations/integrations-in-rancher/fleet-gitops-at-scale/windows-support.md)
```
[the remark this answers is not in the document]
Comment (jtravee) [reply docs/pages-for-subheaders/fleet-gitops-at-scale.md]: Updated per feedback.

**Review by btat — APPROVED** [review]

**Pull request merged**